flexran/securityContext.json

```diff
@@ -0,0 +1,6 @@
+"securityContext": {
+  "privileged": true,
```

Comment:
Are we sure we need this?  Or are we working on figuring out how to not need this?

Reply:
Yes. We need it for mount hostPath.